netbox/extras/jobs.py

```diff
@@ -51,6 +51,9 @@ def run_script(self, script, request, data, commit):
                     logger.warning("Script failed")
                     raise
 
+        except AbortTransaction:
+            # this is already handled above.
+            pass
```

Comment:
This doesn't need to be caught again; we can just avoid re-raising it on line 52.